exchange/websocket/buffer/buffer_test.go

```diff
@@ -742,15 +743,35 @@ func TestGetOrderbook(t *testing.T) {
 	bidLen, err := bufferOb.ob.GetBidLength()
 	require.NoError(t, err)
 
-	if askLen != len(ob.Asks) ||
-		bidLen != len(ob.Bids) ||
-		b.Asset != ob.Asset ||
-		b.Exchange != ob.Exchange ||
-		b.LastUpdateID != ob.LastUpdateID ||
-		b.PriceDuplication != ob.PriceDuplication ||
-		b.Pair != ob.Pair {
-		t.Fatal("data on both books should be the same")
-	}
+	require.Equal(t, askLen, len(ob.Asks))
+	require.Equal(t, bidLen, len(ob.Bids))
+	require.Equal(t, b.Asset, ob.Asset)
+	require.Equal(t, b.Exchange, ob.Exchange)
+	require.Equal(t, b.LastUpdateID, ob.LastUpdateID)
+	require.Equal(t, b.PriceDuplication, ob.PriceDuplication)
+	require.Equal(t, b.Pair, ob.Pair)
```

Comment:
🐒 These don't feel like `require`, and could ideally do with explanations.
```suggestion
	assert.Equal(t, askLen, len(ob.Asks), "Retrieve should get correct number of Asks")
        ... etc ...
```

Reply:
[b737d85](https://github.com/thrasher-corp/gocryptotrader/pull/1863/commits/b737d85b39f1899add14a76098c52d8b24f55b5f)

Done